### ISSUE TITLE
NPM release?

### ISSUE THREAD

**Issue by HenrikBengtsson** [seguid/seguid-javascript#16]
Hello, how far do you think we're from a NPM release?

**Comment by louisabraham** [comment]
Hello, I don't know well the npm ecosystem. I'll ask someone how to improve this.

**Comment by louisabraham** [comment]
I asked @yovanoc and we have a clean release on https://www.npmjs.com/package/seguid

What should I do with https://www.npmjs.com/package/seguid-javascript ? Delete it?

**Comment by HenrikBengtsson** [comment]
Thank you both.

I'm looking at the README; what does "esm" and "cjs" stand for?  I think we should spell them out and give some more information.

**Comment by HenrikBengtsson** [comment]
> What should I do with https://www.npmjs.com/package/seguid-javascript ? Delete it?

I guess it can be deleted

**Comment by louisabraham** [comment]
> I guess it can be deleted

done

> I'm looking at the README; what does "esm" and "cjs" stand for? I think we should spell them out and give some more information.

https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm

it's part of the JS ecosystem mess. People who need them know them, others can just ignore them.

**Comment by HenrikBengtsson** [comment]
> it's part of the JS ecosystem mess. People who need them know them, others can just ignore them.

I disagree; we should avoid adding to/boosting the "mess". They add confusion and ambiguity to people who don't know what they are - it's not much work to spell them out too and add some clarification.

**Comment by louisabraham** [comment]
I mean, spelling them out won't help more than spelling "Tcl" as "Tool Command Language".
what do you suggest?

**Comment by HenrikBengtsson** [comment]
Closing. It's already released on NPM at https://www.npmjs.com/package/seguid
